src/app/flows/fines/fines-mac/fines-mac-fixed-penalty-details/constants/fines-mac-fixed-penalty-details-field-errors.ts

```diff
@@ -0,0 +1,163 @@
+  fm_fp_language_preferences_hearing_language: {},
+  fm_fp_offence_details_notice_number: {
+    required: {
+      message: `Enter Notice number`,
+      priority: 1,
+    },
+    maxlength: {
+      message: `Notice number must be 16 characters or fewer`,
+      priority: 2,
+    },
+    alphabeticalTextPattern: {
+      message: `Notice number must only include letters a to z, numbers, hyphens, spaces and apostrophes`,
+      priority: 3,
+    },
+  },
+  fm_fp_offence_details_offence_type: {},
+  fm_fp_offence_details_date_of_offence: {
+    required: {
+      message: `Enter date of offence `,
+      priority: 1,
+    },
+    invalidDateFormat: {
+      message: `Offence date must be in the format DD/MM/YYYY`,
+      priority: 2,
+    },
+    invalidDate: {
+      message: `Enter a valid offence date`,
+      priority: 2,
+    },
+    invalidFutureDate: {
+      message: `Date of offence must be in the past`,
+      priority: 3,
+    },
+  },
+  fm_fp_offence_details_offence_id: {},
+  fm_fp_offence_details_offence_cjs_code:
+    FINES_MAC_OFFENCE_DETAILS_OFFENCES_FIELD_ERRORS['fm_offence_details_offence_cjs_code'],
+  fm_fp_offence_details_time_of_offence: {
+    invalidTimeFormat: {
+      message: `Enter time of offence in the correct format, such as 02:00 or 14:00`,
+      priority: 1,
+    },
+  },
+  fm_fp_offence_details_place_of_offence: {
+    required: {
+      message: `Enter where the offence took place`,
+      priority: 1,
+    },
+    maxlength: {
+      message: `Place of offence must be 30 characters or fewer`,
+      priority: 2,
+    },
+    alphabeticalTextPattern: {
+      message: `Place of offence must only include letters a to z, numbers, hyphens, spaces and apostrophes`,
+      priority: 3,
+    },
+  },
+  fm_fp_offence_details_amount_imposed: {
+    required: {
+      message: `Enter amount imposed`,
+      priority: 1,
+    },
+    invalidAmountValue: {
+      message: `Enter valid amount`,
+      priority: 2,
+    },
+    invalidAmount: {
+      message: `Amount too long. Enter an amount that is less than 18 numbers before the decimal and 2 or less after`,
+      priority: 3,
+    },
+  },
+  fm_fp_offence_details_vehicle_registration_number: {
+    required: {
+      message: `Enter Registration number`,
+      priority: 1,
+    },
+    maxlength: {
+      message: `Registration number must be 7 characters or fewer `,
+      priority: 1,
+    },
+    alphabeticalTextPattern: {
+      message: `Registration number must only include letters a to z, numbers, hyphens, spaces and apostrophes`,
+      priority: 2,
+    },
+  },
+  fm_fp_offence_details_driving_licence_number: {
+    required: {
+      message: `Enter Driving licence number`,
+      priority: 1,
+    },
+    invalidDrivingLicenceNumber: {
+      message: `Driving licence number must be in a valid format (I.e. first 5 characters are the surname, second 6 are the DOB, 2 characters for the initials and 3 random characters)`,
+      priority: 1,
+    },
```

Comment:
This should be priority 2
```suggestion
    invalidDrivingLicenceNumber: {
      message: `Driving licence number must be in a valid format (I.e. first 5 characters are the surname, second 6 are the DOB, 2 characters for the initials and 3 random characters)`,
      priority: 2,
    },
```